modules/monitoring/files/usr/local/bin/govuk_uptime_collector

```diff
@@ -58,10 +63,11 @@ def main
   threads = []
 
   environment = ARGV[0]
+  aws = ARGV[1]
```

Comment:
Suspect this will have to be `ARGV[1] == 'true' ? true : false` otherwise the string `"false"` will be truthy.

Reply:
Will it ever be a string? It's in Puppet and Hieradata as a boolean. But OK, I've amended.

Reply:
Yeah, annoyingly it will be a string as it comes from https://github.com/alphagov/govuk-puppet/pull/6823/files#diff-3c14b0dbebef6ce25a9e337b66b257fdR9 which passes it in as a normal command line argument.